gooddata-sdk/gooddata_sdk/client.py

```diff
@@ -56,6 +59,46 @@ def __init__(
         self._actions_api = apis.ActionsApi(self._api_client)
         self._user_management_api = apis.UserManagementApi(self._api_client)
 
+    class HttpMethod(Enum):
+        GET = auto()
+        POST = auto()
+        PUT = auto()
+        DELETE = auto()
+        PATCH = auto()
+
+    def _do_post_request(
+        self,
+        data: bytes,
+        endpoint: str,
+        content_type: str,
+    ):
+        try:
+            response = requests.post(
+                url=f"{self._hostname}{endpoint}",
+                headers={
+                    "Content-Type": content_type,
+                    "Authorization": f"Bearer {self._token}",
+                },
+                data=data,
+            )
+        except requests.exceptions.HTTPError as http_err:
+            print(f"HTTP error occurred: {http_err}")
+            print(f"Status Code: {response.status_code}")
+            print(f"Error Details (raw):  {response.text}")
+            print(f"Headers:, {response.headers}")
```

Comment:
I am still trying to figure out this try-catch, so I propose to remove it. If this request fails, the user can't implement any logic based on prints.